docs/swagger/spec.yml

```diff
@@ -843,22 +1113,63 @@ definitions:
         format: uri
         description: Link to User who most recently modified the object
```

Comment:
"Id of User who most recently modified the object"
Links implies a hyperlink.

Same for line 1110